front/style/pages/_referent.scss

```diff
@@ -1,3 +1,16 @@
+.remark {
```

Comment:
We're using the BEM styleguide for CSS. So you shouldn't just add a class on its own. More here : http://getbem.com/naming/